### DISCUSSION TITLE
What is the equivalent of onYouTubeIframeAPIReady in Vimeo?

### DISCUSSION THREAD

**Discussion by marcelobitorical** [vimeo/player.js#815] [Q&A]
👋 Hi, Vimeo community!
I'm searching for a function that is called whenever the API is ready. Like `onYouTubeIframeAPIReady`:

> onYouTubeIframeAPIReady – The API will call this function when the page has finished downloading the JavaScript for the player API, which enables you to then use the API on your page. 

Or is there a technique to achieve something like that?

Thanks in advance 😄

**Comment by rkrishnan8594** [comment]
Hi @marcelobitorical - would using the event `loaded` cover your use case?

```
<div id="player"></div>

<script src="https://player.vimeo.com/api/player.js"></script>
<script>
    const options = {
        id: 59777392,
        width: 640,
        loop: true
    };

    const player = new Vimeo.Player('player', options);
    player.on('loaded', () => {
        // Do whatever you need to do here
    })
</script>

**Reply by marcelobitorical** [reply]
Hi @rkrishnan8594! 😊
The `loaded` event fires when a new video is loaded in the player.

I'm looking for something that fires when the API itself is loaded (which is earlier than the `loaded`).

**Reply by marcelobitorical** [reply]
As I see, maybe there's no **onYouTubeIframeAPIReady** equivalent in Vimeo API. 😥
